packages/react-avatar/src/components/Avatar/_tokens.scss

```diff
@@ -158,3 +158,41 @@ $shadow-16-darker: 0px 0px 4px $color-default-shadowAmbientDarker, 0px 6px 16px
 $shadow-16-lighter: 0px 0px 4px $color-default-shadowAmbientLighter, 0px 6px 16px $color-default-shadowKeyLighter;
 $shadow-28: 0px 0px 8px $color-default-shadowAmbientDarker, 0px 14px 28px $color-default-shadowKeyDarker;
 $shadow-64: 0px 0px 8px $color-default-shadowAmbientDarker, 0px 32x 64px $color-default-shadowKeyDarker;
+
+// Animation timing
+$anim-duration50: 50ms;
+$anim-duration100: 100ms;
+$anim-duration150: 150ms;
+$anim-duration200: 200ms;
+$anim-duration300: 300ms;
+$anim-duration400: 400ms;
+$anim-duration500: 500ms;
+
+$anim-ultraFast: $anim-duration50;
+$anim-faster: $anim-duration100;
+$anim-fast: $anim-duration150;
+$anim-normal: $anim-duration200;
+$anim-slow: $anim-duration300;
+$anim-slower: $anim-duration400;
+$anim-ultraSlow: $anim-duration500;
+
+// Animation curves
+$anim-decelerateMax: cubic-bezier(0.00,0.00,0.00,1.00);

```

Comment:
Seems like we'd probably want to move this to @fluentui/theme.

Reply:
Yeah eventually all of these should come from design tokens via the theme. I didn't want that work to block me here, so I just added them here for now until the rest of that is figured out.